### ISSUE TITLE
testing

### ISSUE THREAD

**Issue by reynoldsnlp** [reynoldsnlp/flair#24]
We need automated testing. Let's use this thread to discuss/plan/prioritize.

**Comment by mjbriggs** [comment]
@jhveem could you send me some of the arabic texts you used to test the accuracy of your model? Ideally I would have one for each readability level

**Comment by jhveem** [comment]
Yeah, I'm about to hop on a plane but I'll try to send it this evening.

**Comment by mjbriggs** [comment]
Okay thanks

**Comment by mjbriggs** [comment]
@jhveem could you push your test files into master branch? They would go in src/test/resources/testFiles. 
I got the stuff that you emailed to rob but the encoding was changed, so I couldn't use them.

**Comment by mjbriggs** [comment]
As of writing this I have written unit tests for the server.raft package and part of the server.parser package. There are also some integration tests that perform web search operations for each language, document upload operations for english and german, and a raft integration test.
